src/sentry/runner/commands/backup.py

```diff
@@ -447,12 +455,11 @@ def decrypt(
 
     try:
         decrypted = decrypt_encrypted_tarball(src, decryptor)
+        with dest:
+            dest.write(decrypted)
     except DecryptionError as e:
         click.echo(f"Invalid tarball: {str(e)}", err=True)
```

Comment:
I would write this using `try:` `else` since keeping the scope of `try` as small as possible (ideally one line / operation) is a best practice:

```python
try:
    decrypted = ...
except ...:
   ...
else:
    with dest:
        dest.write(decrypted)
```

Reply:
Fixed here and below.